src/meta/expr.lean

```diff
@@ -382,6 +382,12 @@ meta def pi_codomain : expr → expr -- see note [open expressions]
 | (pi n bi d b) := pi_codomain b
 | e             := e
 
+/-- Get the body/value of a lambda-expression.
+  This definition doesn't instantiate bound variables, and therefore produces a term that is open. -/
+meta def lambda_body : expr → expr -- see note [open expressions]
```

Comment:
If you move `Note [open expressions]` to the doc string it will get linkified in the html docs.